CHANGELOG.md

```diff
@@ -1,14 +1,14 @@
 # Changelog
 
-## [1.0.1](https://github.com/joeyparrish/karma-local-wd-launcher/compare/v1.0.1...v1.0.0) (2021-11-15)
+## [1.0.1](https://github.com/shaka-project/trace-anything/compare/v1.0.1...v1.0.0) (2021-11-15)
```

Comment:
Wait, this turns from `karma-local-wd-launcher` to `trace-anything`?

Reply:
Ah, I guess that might have been a typo at first.

Reply:
Yup!  I'm a bit embarrassed that I introduced that little copy-paste bug in the first place.